
<!DOCTYPE html>

<html class="no-js" lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width,initial-scale=1" name="viewport"/>
<link href="https://sierra-moxon.github.io/testviz/Serial/" rel="canonical"/>
<link href="../assets/images/favicon.png" rel="icon"/>
<meta content="mkdocs-1.5.3, mkdocs-material-8.5.10" name="generator"/>
<title>Class: Serial - testviz</title>
<link href="../assets/stylesheets/main.975780f9.min.css" rel="stylesheet"/>
<link href="../assets/stylesheets/palette.2505c338.min.css" rel="stylesheet"/>
<link crossorigin="" href="https://fonts.gstatic.com" rel="preconnect"/>
<link href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&amp;display=fallback" rel="stylesheet"/>
<style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
<link href="../style.css" rel="stylesheet"/>
<script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
</head>
<body data-md-color-accent="None" data-md-color-primary="None" data-md-color-scheme="default" dir="ltr">
<input autocomplete="off" class="md-toggle" data-md-toggle="drawer" id="__drawer" type="checkbox"/>
<input autocomplete="off" class="md-toggle" data-md-toggle="search" id="__search" type="checkbox"/>
<label class="md-overlay" for="__drawer"></label>
<div data-md-component="skip">
<a class="md-skip" href="#class-serial">
          Skip to content
        </a>
</div>
<div data-md-component="announce">
</div>
<header class="md-header" data-md-component="header">
<nav aria-label="Header" class="md-header__inner md-grid">
<a aria-label="testviz" class="md-header__button md-logo" data-md-component="logo" href=".." title="testviz">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"></path></svg>
</a>
<label class="md-header__button md-icon" for="__drawer">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"></path></svg>
</label>
<div class="md-header__title" data-md-component="header-title">
<div class="md-header__ellipsis">
<div class="md-header__topic">
<span class="md-ellipsis">
            testviz
          </span>
</div>
<div class="md-header__topic" data-md-component="header-topic">
<span class="md-ellipsis">
            
              Class: Serial
            
          </span>
</div>
</div>
</div>
<label class="md-header__button md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"></path></svg>
</label>
<div class="md-search" data-md-component="search" role="dialog">
<label class="md-search__overlay" for="__search"></label>
<div class="md-search__inner" role="search">
<form class="md-search__form" name="search">
<input aria-label="Search" autocapitalize="off" autocomplete="off" autocorrect="off" class="md-search__input" data-md-component="search-query" name="query" placeholder="Search" required="" spellcheck="false" type="text"/>
<label class="md-search__icon md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"></path></svg>
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"></path></svg>
</label>
<nav aria-label="Search" class="md-search__options">
<button aria-label="Clear" class="md-search__icon md-icon" tabindex="-1" title="Clear" type="reset">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"></path></svg>
</button>
</nav>
</form>
<div class="md-search__output">
<div class="md-search__scrollwrap" data-md-scrollfix="">
<div class="md-search-result" data-md-component="search-result">
<div class="md-search-result__meta">
            Initializing search
          </div>
<ol class="md-search-result__list"></ol>
</div>
</div>
</div>
</div>
</div>
<div class="md-header__source">
<a class="md-source" data-md-component="source" href="https://github.com/sierra-moxon/testviz" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><!--! Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    GitHub
  </div>
</a>
</div>
</nav>
</header>
<div class="md-container" data-md-component="container">
<main class="md-main" data-md-component="main">
<div class="md-main__inner md-grid">
<div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Navigation" class="md-nav md-nav--primary" data-md-level="0">
<label class="md-nav__title" for="__drawer">
<a aria-label="testviz" class="md-nav__button md-logo" data-md-component="logo" href=".." title="testviz">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"></path></svg>
</a>
    testviz
  </label>
<div class="md-nav__source">
<a class="md-source" data-md-component="source" href="https://github.com/sierra-moxon/testviz" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><!--! Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    GitHub
  </div>
</a>
</div>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="..">
        Index
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../about/">
        About
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../information-resource-registry/">
        Infores
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../association-examples-with-qualifiers/">
        Associations?
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../reading-a-qualifier-based-statement/">
        Qualifiers?
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../curating-the-model/">
        Curating Biolink
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../maintaining-the-model/">
        Maintaining Biolink
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../using-the-modeling-language/">
        Using the LinkML Modeling Language
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../working-with-the-model/">
        Working with Biolink
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../categories.html">
        Visualize Categories
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../predicates.html">
        Visualize Predicates
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../faq/">
        FAQ
      </a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#inheritance">
    Inheritance
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#slots">
    Slots
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#aliases">
    Aliases
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#identifier-and-mapping-information">
    Identifier and Mapping Information
  </a>
<nav aria-label="Identifier and Mapping Information" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#valid-id-prefixes">
    Valid ID Prefixes
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#schema-source">
    Schema Source
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#mappings">
    Mappings
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#linkml-source">
    LinkML Source
  </a>
<nav aria-label="LinkML Source" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#direct">
    Direct
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#induced">
    Induced
  </a>
</li>
</ul>
</nav>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-content" data-md-component="content">
<article class="md-content__inner md-typeset">
<a class="md-content__button md-icon" href="https://github.com/sierra-moxon/testviz/edit/master/docs/Serial.md" title="Edit this page">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25Z"></path></svg>
</a>
<h1 id="class-serial">Class: Serial</h1>
<p><em>This class may rarely be instantiated except if use cases of a given knowledge graph support its utility.</em></p>
<p>URI: <a href="https://w3id.org/biolink/vocab/Serial">biolink:Serial</a></p>
<div class="mermaid"> classDiagram
    class Serial
      Publication &lt;|-- Serial

      Serial : authors

          Serial --|&gt; agent : authors

      Serial : category

      Serial : creation_date

      Serial : description

      Serial : format

      Serial : full_name

      Serial : has_attribute

          Serial --|&gt; attribute : has_attribute

      Serial : id

      Serial : iri

      Serial : iso_abbreviation

      Serial : issue

      Serial : keywords

      Serial : license

      Serial : mesh_terms

      Serial : name

      Serial : pages

      Serial : provided_by

      Serial : rights

      Serial : summary

      Serial : synonym

      Serial : type

      Serial : volume

      Serial : xref


</div>
<h2 id="inheritance">Inheritance</h2>
<ul>
<li><a href="../Entity/">Entity</a><ul>
<li><a href="../NamedThing/">NamedThing</a><ul>
<li><a href="../InformationContentEntity/">InformationContentEntity</a><ul>
<li><a href="../Publication/">Publication</a><ul>
<li><strong>Serial</strong></li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>Name</th>
<th>Cardinality and Range</th>
<th>Description</th>
<th>Inheritance</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../iso_abbreviation/">iso_abbreviation</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Standard abbreviation for periodicals in the International Organization for S...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../volume/">volume</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>volume of a book or music release in a collection/series or a published colle...</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../issue/">issue</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>issue of a newspaper, a scientific journal or magazine for reference purpose</td>
<td>direct</td>
</tr>
<tr>
<td><a href="../authors/">authors</a></td>
<td>0..* <br/> <a href="../Agent/">Agent</a></td>
<td>connects an publication to the list of authors who contributed to the publica...</td>
<td><a href="../Publication/">Publication</a></td>
</tr>
<tr>
<td><a href="../pages/">pages</a></td>
<td>0..* <br/> <a href="../String/">String</a></td>
<td>When a 2-tuple of page numbers are provided, they represent the start and end...</td>
<td><a href="../Publication/">Publication</a></td>
</tr>
<tr>
<td><a href="../summary/">summary</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>executive  summary of a publication</td>
<td><a href="../Publication/">Publication</a></td>
</tr>
<tr>
<td><a href="../keywords/">keywords</a></td>
<td>0..* <br/> <a href="../String/">String</a></td>
<td>keywords tagging a publication</td>
<td><a href="../Publication/">Publication</a></td>
</tr>
<tr>
<td><a href="../mesh_terms/">mesh_terms</a></td>
<td>0..* <br/> <a href="../Uriorcurie/">Uriorcurie</a></td>
<td>mesh terms tagging a publication</td>
<td><a href="../Publication/">Publication</a></td>
</tr>
<tr>
<td><a href="../xref/">xref</a></td>
<td>0..* <br/> <a href="../Uriorcurie/">Uriorcurie</a></td>
<td>A database cross reference or alternative identifier for a NamedThing or edge...</td>
<td><a href="../Publication/">Publication</a>, <a href="../NamedThing/">NamedThing</a></td>
</tr>
<tr>
<td><a href="../license/">license</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td></td>
<td><a href="../InformationContentEntity/">InformationContentEntity</a></td>
</tr>
<tr>
<td><a href="../rights/">rights</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td></td>
<td><a href="../InformationContentEntity/">InformationContentEntity</a></td>
</tr>
<tr>
<td><a href="../format/">format</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td></td>
<td><a href="../InformationContentEntity/">InformationContentEntity</a></td>
</tr>
<tr>
<td><a href="../creation_date/">creation_date</a></td>
<td>0..1 <br/> <a href="../Date/">Date</a></td>
<td>date on which an entity was created</td>
<td><a href="../InformationContentEntity/">InformationContentEntity</a></td>
</tr>
<tr>
<td><a href="../provided_by/">provided_by</a></td>
<td>0..* <br/> <a href="../String/">String</a></td>
<td>The value in this node property represents the knowledge provider that create...</td>
<td><a href="../NamedThing/">NamedThing</a></td>
</tr>
<tr>
<td><a href="../full_name/">full_name</a></td>
<td>0..1 <br/> <a href="../LabelType/">LabelType</a></td>
<td>a long-form human readable name for a thing</td>
<td><a href="../NamedThing/">NamedThing</a></td>
</tr>
<tr>
<td><a href="../synonym/">synonym</a></td>
<td>0..* <br/> <a href="../LabelType/">LabelType</a></td>
<td>Alternate human-readable names for a thing</td>
<td><a href="../NamedThing/">NamedThing</a></td>
</tr>
<tr>
<td><a href="../id/">id</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>Serials (journals) should have industry-standard identifier such as from ISSN</td>
<td><a href="../Entity/">Entity</a></td>
</tr>
<tr>
<td><a href="../iri/">iri</a></td>
<td>0..1 <br/> <a href="../IriType/">IriType</a></td>
<td>An IRI for an entity</td>
<td><a href="../Entity/">Entity</a></td>
</tr>
<tr>
<td><a href="../category/">category</a></td>
<td>1..* <br/> <a href="../CategoryType/">CategoryType</a></td>
<td>Name of the high level ontology class in which this entity is categorized</td>
<td><a href="../Entity/">Entity</a></td>
</tr>
<tr>
<td><a href="../type/">type</a></td>
<td>0..* <br/> <a href="../String/">String</a></td>
<td>Should generally be set to an ontology class defined term for 'serial' or 'jo...</td>
<td><a href="../Entity/">Entity</a></td>
</tr>
<tr>
<td><a href="../name/">name</a></td>
<td>0..1 <br/> <a href="../LabelType/">LabelType</a></td>
<td>the 'title' of the publication is generally recorded in the 'name' property (...</td>
<td><a href="../Entity/">Entity</a></td>
</tr>
<tr>
<td><a href="../description/">description</a></td>
<td>0..1 <br/> <a href="../NarrativeText/">NarrativeText</a></td>
<td>a human-readable description of an entity</td>
<td><a href="../Entity/">Entity</a></td>
</tr>
<tr>
<td><a href="../has_attribute/">has_attribute</a></td>
<td>0..* <br/> <a href="../Attribute/">Attribute</a></td>
<td>connects any entity to an attribute</td>
<td><a href="../Entity/">Entity</a></td>
</tr>
</tbody>
</table>
<h2 id="aliases">Aliases</h2>
<ul>
<li>journal</li>
</ul>
<h2 id="identifier-and-mapping-information">Identifier and Mapping Information</h2>
<h3 id="valid-id-prefixes">Valid ID Prefixes</h3>
<p>Instances of this class <em>should</em> have identifiers with one of the following prefixes:</p>
<ul>
<li>
<p>issn</p>
</li>
<li>
<p>NLMID</p>
</li>
</ul>
<h3 id="schema-source">Schema Source</h3>
<ul>
<li>from schema: https://w3id.org/biolink/biolink-model</li>
</ul>
<h2 id="mappings">Mappings</h2>
<table>
<thead>
<tr>
<th>Mapping Type</th>
<th>Mapped Value</th>
</tr>
</thead>
<tbody>
<tr>
<td>self</td>
<td>biolink:Serial</td>
</tr>
<tr>
<td>native</td>
<td>biolink:Serial</td>
</tr>
</tbody>
</table>
<h2 id="linkml-source">LinkML Source</h2>
<!-- TODO: investigate https://stackoverflow.com/questions/37606292/how-to-create-tabbed-code-blocks-in-mkdocs-or-sphinx -->
<h3 id="direct">Direct</h3>
<details>
<pre><code class="language-yaml">name: serial
id_prefixes:
- issn
- NLMID
description: This class may rarely be instantiated except if use cases of a given
  knowledge graph support its utility.
in_subset:
- model_organism_database
from_schema: https://w3id.org/biolink/biolink-model
aliases:
- journal
is_a: publication
slots:
- iso abbreviation
- volume
- issue
slot_usage:
  id:
    name: id
    description: Serials (journals) should have industry-standard identifier such
      as from ISSN.
    domain_of:
    - ontology class
    - entity
    required: true
  type:
    name: type
    description: Should generally be set to an ontology class defined term for 'serial'
      or 'journal'.
    domain_of:
    - entity

</code></pre>
</details>
<h3 id="induced">Induced</h3>
<details>
<pre><code class="language-yaml">name: serial
id_prefixes:
- issn
- NLMID
description: This class may rarely be instantiated except if use cases of a given
  knowledge graph support its utility.
in_subset:
- model_organism_database
from_schema: https://w3id.org/biolink/biolink-model
aliases:
- journal
is_a: publication
slot_usage:
  id:
    name: id
    description: Serials (journals) should have industry-standard identifier such
      as from ISSN.
    domain_of:
    - ontology class
    - entity
    required: true
  type:
    name: type
    description: Should generally be set to an ontology class defined term for 'serial'
      or 'journal'.
    domain_of:
    - entity
attributes:
  iso abbreviation:
    name: iso abbreviation
    description: Standard abbreviation for periodicals in the International Organization
      for Standardization (ISO) 4 system See https://www.issn.org/services/online-services/access-to-the-ltwa/.
      If the 'published in' property is set, then the iso abbreviation pertains to
      the broader publication context (the journal) within which the given publication
      node is embedded, not the publication itself.
    from_schema: https://w3id.org/biolink/biolink-model
    exact_mappings:
    - WIKIDATA_PROPERTY:P1160
    rank: 1000
    is_a: node property
    domain: publication
    alias: iso_abbreviation
    owner: serial
    domain_of:
    - serial
    - article
    range: string
  volume:
    name: volume
    description: volume of a book or music release in a collection/series or a published
      collection of journal issues in a serial publication
    from_schema: https://w3id.org/biolink/biolink-model
    exact_mappings:
    - WIKIDATA_PROPERTY:P478
    rank: 1000
    is_a: node property
    domain: publication
    alias: volume
    owner: serial
    domain_of:
    - book chapter
    - serial
    - article
    range: string
  issue:
    name: issue
    description: issue of a newspaper, a scientific journal or magazine for reference
      purpose
    from_schema: https://w3id.org/biolink/biolink-model
    exact_mappings:
    - WIKIDATA_PROPERTY:P433
    rank: 1000
    is_a: node property
    domain: publication
    alias: issue
    owner: serial
    domain_of:
    - serial
    - article
    range: string
  authors:
    name: authors
    description: connects an publication to the list of authors who contributed to
      the publication. This property should be a comma-delimited list of author names.
      It is recommended that an author's name be formatted as "surname, firstname
      initial.".   Note that this property is a node annotation expressing the citation
      list of authorship which might typically otherwise be more completely documented
      in biolink:PublicationToProviderAssociation defined edges which point to full
      details about an author and possibly, some qualifiers which clarify the specific
      status of a given author in the publication.
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: node property
    domain: publication
    multivalued: true
    alias: authors
    owner: serial
    domain_of:
    - publication
    range: agent
  pages:
    name: pages
    description: When a 2-tuple of page numbers are provided, they represent the start
      and end page of the publication within its parent publication context. For books,
      this may be set to the total number of pages of the book.
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: node property
    domain: publication
    multivalued: true
    alias: pages
    owner: serial
    domain_of:
    - publication
    range: string
  summary:
    name: summary
    description: executive  summary of a publication
    from_schema: https://w3id.org/biolink/biolink-model
    aliases:
    - abstract
    exact_mappings:
    - dct:abstract
    - WIKIDATA:Q333291
    rank: 1000
    is_a: node property
    domain: publication
    alias: summary
    owner: serial
    domain_of:
    - publication
    range: string
  keywords:
    name: keywords
    description: keywords tagging a publication
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: node property
    domain: publication
    multivalued: true
    alias: keywords
    owner: serial
    domain_of:
    - publication
    range: string
  mesh terms:
    name: mesh terms
    description: mesh terms tagging a publication
    from_schema: https://w3id.org/biolink/biolink-model
    exact_mappings:
    - dcid:MeSHTerm
    rank: 1000
    is_a: node property
    values_from:
    - MESH
    domain: publication
    multivalued: true
    alias: mesh_terms
    owner: serial
    domain_of:
    - publication
    range: uriorcurie
  xref:
    name: xref
    description: A database cross reference or alternative identifier for a NamedThing
      or edge between two  NamedThings.  This property should point to a database
      record or webpage that supports the existence of the edge, or  gives more detail
      about the edge. This property can be used on a node or edge to provide multiple
      URIs or CURIE cross references.
    in_subset:
    - translator_minimal
    from_schema: https://w3id.org/biolink/biolink-model
    aliases:
    - dbxref
    - Dbxref
    - DbXref
    - record_url
    - source_record_urls
    narrow_mappings:
    - gff3:Dbxref
    - gpi:DB_Xrefs
    rank: 1000
    domain: named thing
    multivalued: true
    alias: xref
    owner: serial
    domain_of:
    - named thing
    - publication
    - retrieval source
    - gene
    - gene product mixin
    range: uriorcurie
  license:
    name: license
    from_schema: https://w3id.org/biolink/biolink-model
    exact_mappings:
    - dct:license
    narrow_mappings:
    - WIKIDATA_PROPERTY:P275
    rank: 1000
    is_a: node property
    domain: information content entity
    alias: license
    owner: serial
    domain_of:
    - information content entity
    range: string
  rights:
    name: rights
    from_schema: https://w3id.org/biolink/biolink-model
    exact_mappings:
    - dct:rights
    rank: 1000
    is_a: node property
    domain: information content entity
    alias: rights
    owner: serial
    domain_of:
    - information content entity
    range: string
  format:
    name: format
    from_schema: https://w3id.org/biolink/biolink-model
    exact_mappings:
    - dct:format
    - WIKIDATA_PROPERTY:P2701
    rank: 1000
    is_a: node property
    domain: information content entity
    alias: format
    owner: serial
    domain_of:
    - information content entity
    range: string
  creation date:
    name: creation date
    description: date on which an entity was created. This can be applied to nodes
      or edges
    from_schema: https://w3id.org/biolink/biolink-model
    aliases:
    - publication date
    exact_mappings:
    - dct:createdOn
    - WIKIDATA_PROPERTY:P577
    rank: 1000
    is_a: node property
    domain: named thing
    alias: creation_date
    owner: serial
    domain_of:
    - information content entity
    range: date
  provided by:
    name: provided by
    description: The value in this node property represents the knowledge provider
      that created or assembled the node and all of its attributes.  Used internally
      to represent how a particular node made its way into a knowledge provider or
      graph.
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: node property
    domain: named thing
    multivalued: true
    alias: provided_by
    owner: serial
    domain_of:
    - named thing
    range: string
  full name:
    name: full name
    description: a long-form human readable name for a thing
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: node property
    domain: named thing
    alias: full_name
    owner: serial
    domain_of:
    - named thing
    range: label type
  synonym:
    name: synonym
    description: Alternate human-readable names for a thing
    in_subset:
    - translator_minimal
    from_schema: https://w3id.org/biolink/biolink-model
    aliases:
    - alias
    narrow_mappings:
    - skos:altLabel
    - gff3:Alias
    - AGRKB:synonyms
    - gpi:DB_Object_Synonyms
    - HANCESTRO:0330
    - IAO:0000136
    - RXNORM:has_tradename
    rank: 1000
    is_a: node property
    domain: named thing
    multivalued: true
    alias: synonym
    owner: serial
    domain_of:
    - named thing
    - gene product mixin
    range: label type
  id:
    name: id
    description: Serials (journals) should have industry-standard identifier such
      as from ISSN.
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    domain: entity
    identifier: true
    alias: id
    owner: serial
    domain_of:
    - ontology class
    - entity
    range: string
    required: true
  iri:
    name: iri
    description: An IRI for an entity. This is determined by the id using expansion
      rules.
    in_subset:
    - translator_minimal
    - samples
    from_schema: https://w3id.org/biolink/biolink-model
    exact_mappings:
    - WIKIDATA_PROPERTY:P854
    rank: 1000
    alias: iri
    owner: serial
    domain_of:
    - attribute
    - entity
    range: iri type
  category:
    name: category
    description: "Name of the high level ontology class in which this entity is categorized.\
      \ Corresponds to the label for the biolink entity type class.\n * In a neo4j\
      \ database this MAY correspond to the neo4j label tag.\n * In an RDF database\
      \ it should be a biolink model class URI.\nThis field is multi-valued. It should\
      \ include values for ancestors of the biolink class; for example, a protein\
      \ such as Shh would have category values `biolink:Protein`, `biolink:GeneProduct`,\
      \ `biolink:MolecularEntity`, ...\nIn an RDF database, nodes will typically have\
      \ an rdf:type triples. This can be to the most specific biolink class, or potentially\
      \ to a class more specific than something in biolink. For example, a sequence\
      \ feature `f` may have a rdf:type assertion to a SO class such as TF_binding_site,\
      \ which is more specific than anything in biolink. Here we would have categories\
      \ {biolink:GenomicEntity, biolink:MolecularEntity, biolink:NamedThing}"
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: type
    domain: entity
    multivalued: true
    designates_type: true
    alias: category
    owner: serial
    domain_of:
    - entity
    is_class_field: true
    range: category type
    required: true
    pattern: ^biolink:[A-Z][A-Za-z]+$
  type:
    name: type
    description: Should generally be set to an ontology class defined term for 'serial'
      or 'journal'.
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    domain: entity
    slot_uri: rdf:type
    multivalued: true
    alias: type
    owner: serial
    domain_of:
    - entity
    range: string
  name:
    name: name
    description: the 'title' of the publication is generally recorded in the 'name'
      property (inherited from NamedThing). The field name 'title' is now also tagged
      as an acceptable alias for the node property 'name' (just in case).
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    domain: entity
    slot_uri: rdfs:label
    alias: name
    owner: serial
    domain_of:
    - attribute
    - entity
    - macromolecular machine mixin
    range: label type
  description:
    name: description
    description: a human-readable description of an entity
    in_subset:
    - translator_minimal
    from_schema: https://w3id.org/biolink/biolink-model
    aliases:
    - definition
    exact_mappings:
    - IAO:0000115
    - skos:definitions
    narrow_mappings:
    - gff3:Description
    rank: 1000
    slot_uri: dct:description
    alias: description
    owner: serial
    domain_of:
    - entity
    range: narrative text
  has attribute:
    name: has attribute
    description: connects any entity to an attribute
    in_subset:
    - samples
    from_schema: https://w3id.org/biolink/biolink-model
    exact_mappings:
    - SIO:000008
    close_mappings:
    - OBI:0001927
    narrow_mappings:
    - OBAN:association_has_subject_property
    - OBAN:association_has_object_property
    - CPT:has_possibly_included_panel_element
    - DRUGBANK:category
    - EFO:is_executed_in
    - HANCESTRO:0301
    - LOINC:has_action_guidance
    - LOINC:has_adjustment
    - LOINC:has_aggregation_view
    - LOINC:has_approach_guidance
    - LOINC:has_divisor
    - LOINC:has_exam
    - LOINC:has_method
    - LOINC:has_modality_subtype
    - LOINC:has_object_guidance
    - LOINC:has_scale
    - LOINC:has_suffix
    - LOINC:has_time_aspect
    - LOINC:has_time_modifier
    - LOINC:has_timing_of
    - NCIT:R88
    - NCIT:eo_disease_has_property_or_attribute
    - NCIT:has_data_element
    - NCIT:has_pharmaceutical_administration_method
    - NCIT:has_pharmaceutical_basic_dose_form
    - NCIT:has_pharmaceutical_intended_site
    - NCIT:has_pharmaceutical_release_characteristics
    - NCIT:has_pharmaceutical_state_of_matter
    - NCIT:has_pharmaceutical_transformation
    - NCIT:is_qualified_by
    - NCIT:qualifier_applies_to
    - NCIT:role_has_domain
    - NCIT:role_has_range
    - INO:0000154
    - HANCESTRO:0308
    - OMIM:has_inheritance_type
    - orphanet:C016
    - orphanet:C017
    - RO:0000053
    - RO:0000086
    - RO:0000087
    - SNOMED:has_access
    - SNOMED:has_clinical_course
    - SNOMED:has_count_of_base_of_active_ingredient
    - SNOMED:has_dose_form_administration_method
    - SNOMED:has_dose_form_release_characteristic
    - SNOMED:has_dose_form_transformation
    - SNOMED:has_finding_context
    - SNOMED:has_finding_informer
    - SNOMED:has_inherent_attribute
    - SNOMED:has_intent
    - SNOMED:has_interpretation
    - SNOMED:has_laterality
    - SNOMED:has_measurement_method
    - SNOMED:has_method
    - SNOMED:has_priority
    - SNOMED:has_procedure_context
    - SNOMED:has_process_duration
    - SNOMED:has_property
    - SNOMED:has_revision_status
    - SNOMED:has_scale_type
    - SNOMED:has_severity
    - SNOMED:has_specimen
    - SNOMED:has_state_of_matter
    - SNOMED:has_subject_relationship_context
    - SNOMED:has_surgical_approach
    - SNOMED:has_technique
    - SNOMED:has_temporal_context
    - SNOMED:has_time_aspect
    - SNOMED:has_units
    - UMLS:has_structural_class
    - UMLS:has_supported_concept_property
    - UMLS:has_supported_concept_relationship
    - UMLS:may_be_qualified_by
    rank: 1000
    domain: entity
    multivalued: true
    alias: has_attribute
    owner: serial
    domain_of:
    - entity
    range: attribute

</code></pre>
</details>
</article>
</div>
<script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var tab,labels=set.querySelector(".tabbed-labels");for(tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>
</div>
</main>
<footer class="md-footer">
<div class="md-footer-meta md-typeset">
<div class="md-footer-meta__inner md-grid">
<div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" rel="noopener" target="_blank">
      Material for MkDocs
    </a>
</div>
</div>
</div>
</footer>
</div>
<div class="md-dialog" data-md-component="dialog">
<div class="md-dialog__inner md-typeset"></div>
</div>
<script id="__config" type="application/json">{"base": "..", "features": ["content.tabs.link"], "search": "../assets/javascripts/workers/search.16e2a7d4.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version.title": "Select version"}}</script>
<script src="../assets/javascripts/bundle.5a2dcb6a.min.js"></script>
<script src="https://unpkg.com/mermaid@8.8.0/dist/mermaid.min.js"></script><script>mermaid.initialize({});</script></body>
</html>