
<!DOCTYPE html>

<html class="no-js" lang="en">
<head>
<meta charset="utf-8"/>
<meta content="width=device-width,initial-scale=1" name="viewport"/>
<link href="https://sierra-moxon.github.io/testviz/GeneAsAModelOfDiseaseAssociation/" rel="canonical"/>
<link href="../assets/images/favicon.png" rel="icon"/>
<meta content="mkdocs-1.5.3, mkdocs-material-8.5.10" name="generator"/>
<title>Class: GeneAsAModelOfDiseaseAssociation - testviz</title>
<link href="../assets/stylesheets/main.975780f9.min.css" rel="stylesheet"/>
<link href="../assets/stylesheets/palette.2505c338.min.css" rel="stylesheet"/>
<link crossorigin="" href="https://fonts.gstatic.com" rel="preconnect"/>
<link href="https://fonts.googleapis.com/css?family=Roboto:300,300i,400,400i,700,700i%7CRoboto+Mono:400,400i,700,700i&amp;display=fallback" rel="stylesheet"/>
<style>:root{--md-text-font:"Roboto";--md-code-font:"Roboto Mono"}</style>
<script>__md_scope=new URL("..",location),__md_hash=e=>[...e].reduce((e,_)=>(e<<5)-e+_.charCodeAt(0),0),__md_get=(e,_=localStorage,t=__md_scope)=>JSON.parse(_.getItem(t.pathname+"."+e)),__md_set=(e,_,t=localStorage,a=__md_scope)=>{try{t.setItem(a.pathname+"."+e,JSON.stringify(_))}catch(e){}}</script>
</head>
<body data-md-color-accent="None" data-md-color-primary="None" data-md-color-scheme="default" dir="ltr">
<input autocomplete="off" class="md-toggle" data-md-toggle="drawer" id="__drawer" type="checkbox"/>
<input autocomplete="off" class="md-toggle" data-md-toggle="search" id="__search" type="checkbox"/>
<label class="md-overlay" for="__drawer"></label>
<div data-md-component="skip">
<a class="md-skip" href="#class-geneasamodelofdiseaseassociation">
          Skip to content
        </a>
</div>
<div data-md-component="announce">
</div>
<header class="md-header" data-md-component="header">
<nav aria-label="Header" class="md-header__inner md-grid">
<a aria-label="testviz" class="md-header__button md-logo" data-md-component="logo" href=".." title="testviz">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"></path></svg>
</a>
<label class="md-header__button md-icon" for="__drawer">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M3 6h18v2H3V6m0 5h18v2H3v-2m0 5h18v2H3v-2Z"></path></svg>
</label>
<div class="md-header__title" data-md-component="header-title">
<div class="md-header__ellipsis">
<div class="md-header__topic">
<span class="md-ellipsis">
            testviz
          </span>
</div>
<div class="md-header__topic" data-md-component="header-topic">
<span class="md-ellipsis">
            
              Class: GeneAsAModelOfDiseaseAssociation
            
          </span>
</div>
</div>
</div>
<label class="md-header__button md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"></path></svg>
</label>
<div class="md-search" data-md-component="search" role="dialog">
<label class="md-search__overlay" for="__search"></label>
<div class="md-search__inner" role="search">
<form class="md-search__form" name="search">
<input aria-label="Search" autocapitalize="off" autocomplete="off" autocorrect="off" class="md-search__input" data-md-component="search-query" name="query" placeholder="Search" required="" spellcheck="false" type="text"/>
<label class="md-search__icon md-icon" for="__search">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M9.5 3A6.5 6.5 0 0 1 16 9.5c0 1.61-.59 3.09-1.56 4.23l.27.27h.79l5 5-1.5 1.5-5-5v-.79l-.27-.27A6.516 6.516 0 0 1 9.5 16 6.5 6.5 0 0 1 3 9.5 6.5 6.5 0 0 1 9.5 3m0 2C7 5 5 7 5 9.5S7 14 9.5 14 14 12 14 9.5 12 5 9.5 5Z"></path></svg>
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20 11v2H8l5.5 5.5-1.42 1.42L4.16 12l7.92-7.92L13.5 5.5 8 11h12Z"></path></svg>
</label>
<nav aria-label="Search" class="md-search__options">
<button aria-label="Clear" class="md-search__icon md-icon" tabindex="-1" title="Clear" type="reset">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41Z"></path></svg>
</button>
</nav>
</form>
<div class="md-search__output">
<div class="md-search__scrollwrap" data-md-scrollfix="">
<div class="md-search-result" data-md-component="search-result">
<div class="md-search-result__meta">
            Initializing search
          </div>
<ol class="md-search-result__list"></ol>
</div>
</div>
</div>
</div>
</div>
<div class="md-header__source">
<a class="md-source" data-md-component="source" href="https://github.com/sierra-moxon/testviz" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><!--! Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    GitHub
  </div>
</a>
</div>
</nav>
</header>
<div class="md-container" data-md-component="container">
<main class="md-main" data-md-component="main">
<div class="md-main__inner md-grid">
<div class="md-sidebar md-sidebar--primary" data-md-component="sidebar" data-md-type="navigation">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Navigation" class="md-nav md-nav--primary" data-md-level="0">
<label class="md-nav__title" for="__drawer">
<a aria-label="testviz" class="md-nav__button md-logo" data-md-component="logo" href=".." title="testviz">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M12 8a3 3 0 0 0 3-3 3 3 0 0 0-3-3 3 3 0 0 0-3 3 3 3 0 0 0 3 3m0 3.54C9.64 9.35 6.5 8 3 8v11c3.5 0 6.64 1.35 9 3.54 2.36-2.19 5.5-3.54 9-3.54V8c-3.5 0-6.64 1.35-9 3.54Z"></path></svg>
</a>
    testviz
  </label>
<div class="md-nav__source">
<a class="md-source" data-md-component="source" href="https://github.com/sierra-moxon/testviz" title="Go to repository">
<div class="md-source__icon md-icon">
<svg viewbox="0 0 448 512" xmlns="http://www.w3.org/2000/svg"><!--! Font Awesome Free 6.2.1 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License) Copyright 2022 Fonticons, Inc.--><path d="M439.55 236.05 244 40.45a28.87 28.87 0 0 0-40.81 0l-40.66 40.63 51.52 51.52c27.06-9.14 52.68 16.77 43.39 43.68l49.66 49.66c34.23-11.8 61.18 31 35.47 56.69-26.49 26.49-70.21-2.87-56-37.34L240.22 199v121.85c25.3 12.54 22.26 41.85 9.08 55a34.34 34.34 0 0 1-48.55 0c-17.57-17.6-11.07-46.91 11.25-56v-123c-20.8-8.51-24.6-30.74-18.64-45L142.57 101 8.45 235.14a28.86 28.86 0 0 0 0 40.81l195.61 195.6a28.86 28.86 0 0 0 40.8 0l194.69-194.69a28.86 28.86 0 0 0 0-40.81z"></path></svg>
</div>
<div class="md-source__repository">
    GitHub
  </div>
</a>
</div>
<ul class="md-nav__list" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="..">
        Index
      </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="../about/">
        About
      </a>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-sidebar md-sidebar--secondary" data-md-component="sidebar" data-md-type="toc">
<div class="md-sidebar__scrollwrap">
<div class="md-sidebar__inner">
<nav aria-label="Table of contents" class="md-nav md-nav--secondary">
<label class="md-nav__title" for="__toc">
<span class="md-nav__icon md-icon"></span>
      Table of contents
    </label>
<ul class="md-nav__list" data-md-component="toc" data-md-scrollfix="">
<li class="md-nav__item">
<a class="md-nav__link" href="#inheritance">
    Inheritance
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#slots">
    Slots
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#identifier-and-mapping-information">
    Identifier and Mapping Information
  </a>
<nav aria-label="Identifier and Mapping Information" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#schema-source">
    Schema Source
  </a>
</li>
</ul>
</nav>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#mappings">
    Mappings
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#linkml-source">
    LinkML Source
  </a>
<nav aria-label="LinkML Source" class="md-nav">
<ul class="md-nav__list">
<li class="md-nav__item">
<a class="md-nav__link" href="#direct">
    Direct
  </a>
</li>
<li class="md-nav__item">
<a class="md-nav__link" href="#induced">
    Induced
  </a>
</li>
</ul>
</nav>
</li>
</ul>
</nav>
</div>
</div>
</div>
<div class="md-content" data-md-component="content">
<article class="md-content__inner md-typeset">
<a class="md-content__button md-icon" href="https://github.com/sierra-moxon/testviz/edit/master/docs/GeneAsAModelOfDiseaseAssociation.md" title="Edit this page">
<svg viewbox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M20.71 7.04c.39-.39.39-1.04 0-1.41l-2.34-2.34c-.37-.39-1.02-.39-1.41 0l-1.84 1.83 3.75 3.75M3 17.25V21h3.75L17.81 9.93l-3.75-3.75L3 17.25Z"></path></svg>
</a>
<h1 id="class-geneasamodelofdiseaseassociation">Class: GeneAsAModelOfDiseaseAssociation</h1>
<p>URI: <a href="https://w3id.org/biolink/vocab/GeneAsAModelOfDiseaseAssociation">biolink:GeneAsAModelOfDiseaseAssociation</a></p>
<div class="mermaid"> classDiagram
    class GeneAsAModelOfDiseaseAssociation
      ModelToDiseaseAssociationMixin &lt;|-- GeneAsAModelOfDiseaseAssociation
      EntityToDiseaseAssociationMixin &lt;|-- GeneAsAModelOfDiseaseAssociation
      GeneToDiseaseAssociation &lt;|-- GeneAsAModelOfDiseaseAssociation

      GeneAsAModelOfDiseaseAssociation : aggregator_knowledge_source

      GeneAsAModelOfDiseaseAssociation : category

      GeneAsAModelOfDiseaseAssociation : description

      GeneAsAModelOfDiseaseAssociation : frequency_qualifier

      GeneAsAModelOfDiseaseAssociation : has_attribute

          GeneAsAModelOfDiseaseAssociation --|&gt; attribute : has_attribute

      GeneAsAModelOfDiseaseAssociation : has_count

      GeneAsAModelOfDiseaseAssociation : has_evidence

          GeneAsAModelOfDiseaseAssociation --|&gt; evidence type : has_evidence

      GeneAsAModelOfDiseaseAssociation : has_percentage

      GeneAsAModelOfDiseaseAssociation : has_quotient

      GeneAsAModelOfDiseaseAssociation : has_total

      GeneAsAModelOfDiseaseAssociation : id

      GeneAsAModelOfDiseaseAssociation : iri

      GeneAsAModelOfDiseaseAssociation : knowledge_source

      GeneAsAModelOfDiseaseAssociation : name

      GeneAsAModelOfDiseaseAssociation : negated

      GeneAsAModelOfDiseaseAssociation : object

          GeneAsAModelOfDiseaseAssociation --|&gt; disease : object

      GeneAsAModelOfDiseaseAssociation : object_category

          GeneAsAModelOfDiseaseAssociation --|&gt; ontology class : object_category

      GeneAsAModelOfDiseaseAssociation : object_category_closure

          GeneAsAModelOfDiseaseAssociation --|&gt; ontology class : object_category_closure

      GeneAsAModelOfDiseaseAssociation : object_closure

      GeneAsAModelOfDiseaseAssociation : object_direction_qualifier

          GeneAsAModelOfDiseaseAssociation --|&gt; DirectionQualifierEnum : object_direction_qualifier

      GeneAsAModelOfDiseaseAssociation : object_label_closure

      GeneAsAModelOfDiseaseAssociation : object_namespace

      GeneAsAModelOfDiseaseAssociation : onset_qualifier

          GeneAsAModelOfDiseaseAssociation --|&gt; onset : onset_qualifier

      GeneAsAModelOfDiseaseAssociation : original_object

      GeneAsAModelOfDiseaseAssociation : original_predicate

      GeneAsAModelOfDiseaseAssociation : original_subject

      GeneAsAModelOfDiseaseAssociation : predicate

      GeneAsAModelOfDiseaseAssociation : primary_knowledge_source

      GeneAsAModelOfDiseaseAssociation : publications

          GeneAsAModelOfDiseaseAssociation --|&gt; publication : publications

      GeneAsAModelOfDiseaseAssociation : qualifiers

          GeneAsAModelOfDiseaseAssociation --|&gt; ontology class : qualifiers

      GeneAsAModelOfDiseaseAssociation : retrieval_source_ids

          GeneAsAModelOfDiseaseAssociation --|&gt; retrieval source : retrieval_source_ids

      GeneAsAModelOfDiseaseAssociation : severity_qualifier

          GeneAsAModelOfDiseaseAssociation --|&gt; severity value : severity_qualifier

      GeneAsAModelOfDiseaseAssociation : sex_qualifier

          GeneAsAModelOfDiseaseAssociation --|&gt; biological sex : sex_qualifier

      GeneAsAModelOfDiseaseAssociation : subject

          GeneAsAModelOfDiseaseAssociation --|&gt; gene or gene product : subject

      GeneAsAModelOfDiseaseAssociation : subject_aspect_qualifier

          GeneAsAModelOfDiseaseAssociation --|&gt; GeneOrGeneProductOrChemicalEntityAspectEnum : subject_aspect_qualifier

      GeneAsAModelOfDiseaseAssociation : subject_category

          GeneAsAModelOfDiseaseAssociation --|&gt; ontology class : subject_category

      GeneAsAModelOfDiseaseAssociation : subject_category_closure

          GeneAsAModelOfDiseaseAssociation --|&gt; ontology class : subject_category_closure

      GeneAsAModelOfDiseaseAssociation : subject_closure

      GeneAsAModelOfDiseaseAssociation : subject_label_closure

      GeneAsAModelOfDiseaseAssociation : subject_namespace

      GeneAsAModelOfDiseaseAssociation : timepoint

      GeneAsAModelOfDiseaseAssociation : type


</div>
<h2 id="inheritance">Inheritance</h2>
<ul>
<li><a href="../Entity/">Entity</a><ul>
<li><a href="../Association/">Association</a><ul>
<li><a href="../GeneToDiseaseOrPhenotypicFeatureAssociation/">GeneToDiseaseOrPhenotypicFeatureAssociation</a> [ <a href="../EntityToPhenotypicFeatureAssociationMixin/">EntityToPhenotypicFeatureAssociationMixin</a> <a href="../GeneToEntityAssociationMixin/">GeneToEntityAssociationMixin</a>]<ul>
<li><a href="../GeneToDiseaseAssociation/">GeneToDiseaseAssociation</a> [ <a href="../EntityToDiseaseAssociationMixin/">EntityToDiseaseAssociationMixin</a> <a href="../GeneToEntityAssociationMixin/">GeneToEntityAssociationMixin</a>]<ul>
<li><strong>GeneAsAModelOfDiseaseAssociation</strong> [ <a href="../ModelToDiseaseAssociationMixin/">ModelToDiseaseAssociationMixin</a> <a href="../EntityToDiseaseAssociationMixin/">EntityToDiseaseAssociationMixin</a>]</li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
</ul>
</li>
</ul>
<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>Name</th>
<th>Cardinality and Range</th>
<th>Description</th>
<th>Inheritance</th>
</tr>
</thead>
<tbody>
<tr>
<td><a href="../subject_aspect_qualifier/">subject_aspect_qualifier</a></td>
<td>0..1 <br/> <a href="../GeneOrGeneProductOrChemicalEntityAspectEnum/">GeneOrGeneProductOrChemicalEntityAspectEnum</a></td>
<td></td>
<td><a href="../GeneToDiseaseOrPhenotypicFeatureAssociation/">GeneToDiseaseOrPhenotypicFeatureAssociation</a></td>
</tr>
<tr>
<td><a href="../object_direction_qualifier/">object_direction_qualifier</a></td>
<td>0..1 <br/> <a href="../DirectionQualifierEnum/">DirectionQualifierEnum</a></td>
<td></td>
<td><a href="../GeneToDiseaseOrPhenotypicFeatureAssociation/">GeneToDiseaseOrPhenotypicFeatureAssociation</a></td>
</tr>
<tr>
<td><a href="../sex_qualifier/">sex_qualifier</a></td>
<td>0..1 <br/> <a href="../BiologicalSex/">BiologicalSex</a></td>
<td>a qualifier used in a phenotypic association to state whether the association...</td>
<td><a href="../EntityToPhenotypicFeatureAssociationMixin/">EntityToPhenotypicFeatureAssociationMixin</a></td>
</tr>
<tr>
<td><a href="../subject/">subject</a></td>
<td>1..1 <br/> <a href="../GeneOrGeneProduct/">GeneOrGeneProduct</a></td>
<td>A gene that has a role in modeling the disease</td>
<td><a href="../Association/">Association</a></td>
</tr>
<tr>
<td><a href="../predicate/">predicate</a></td>
<td>1..1 <br/> <a href="../PredicateType/">PredicateType</a></td>
<td>The relationship to the disease</td>
<td><a href="../Association/">Association</a></td>
</tr>
<tr>
<td><a href="../object/">object</a></td>
<td>1..1 <br/> <a href="../Disease/">Disease</a></td>
<td>disease</td>
<td><a href="../Association/">Association</a></td>
</tr>
<tr>
<td><a href="../negated/">negated</a></td>
<td>0..1 <br/> <a href="../Boolean/">Boolean</a></td>
<td>if set to true, then the association is negated i</td>
<td><a href="../Association/">Association</a></td>
</tr>
<tr>
<td><a href="../qualifiers/">qualifiers</a></td>
<td>0..* <br/> <a href="../OntologyClass/">OntologyClass</a></td>
<td>connects an association to qualifiers that modify or qualify the meaning of t...</td>
<td><a href="../Association/">Association</a></td>
</tr>
<tr>
<td><a href="../publications/">publications</a></td>
<td>0..* <br/> <a href="../Publication/">Publication</a></td>
<td>One or more publications that report the statement expressed in an  Associati...</td>
<td><a href="../Association/">Association</a></td>
</tr>
<tr>
<td><a href="../has_evidence/">has_evidence</a></td>
<td>0..* <br/> <a href="../EvidenceType/">EvidenceType</a></td>
<td>connects an association to an instance of supporting evidence</td>
<td><a href="../Association/">Association</a></td>
</tr>
<tr>
<td><a href="../knowledge_source/">knowledge_source</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>An Information Resource from which the knowledge expressed in an Association ...</td>
<td><a href="../Association/">Association</a></td>
</tr>
<tr>
<td><a href="../primary_knowledge_source/">primary_knowledge_source</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>The most upstream source of the knowledge expressed in an Association that an...</td>
<td><a href="../Association/">Association</a></td>
</tr>
<tr>
<td><a href="../aggregator_knowledge_source/">aggregator_knowledge_source</a></td>
<td>0..* <br/> <a href="../String/">String</a></td>
<td>An intermediate aggregator resource from which knowledge expressed in an Asso...</td>
<td><a href="../Association/">Association</a></td>
</tr>
<tr>
<td><a href="../timepoint/">timepoint</a></td>
<td>0..1 <br/> <a href="../TimeType/">TimeType</a></td>
<td>a point in time</td>
<td><a href="../Association/">Association</a></td>
</tr>
<tr>
<td><a href="../original_subject/">original_subject</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>used to hold the original subject of a relation (or predicate) that an extern...</td>
<td><a href="../Association/">Association</a></td>
</tr>
<tr>
<td><a href="../original_predicate/">original_predicate</a></td>
<td>0..1 <br/> <a href="../Uriorcurie/">Uriorcurie</a></td>
<td>used to hold the original relation/predicate that an external knowledge sourc...</td>
<td><a href="../Association/">Association</a></td>
</tr>
<tr>
<td><a href="../original_object/">original_object</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>used to hold the original object of a relation (or predicate) that an externa...</td>
<td><a href="../Association/">Association</a></td>
</tr>
<tr>
<td><a href="../subject_category/">subject_category</a></td>
<td>0..1 <br/> <a href="../OntologyClass/">OntologyClass</a></td>
<td>Used to hold the biolink class/category of an association</td>
<td><a href="../Association/">Association</a></td>
</tr>
<tr>
<td><a href="../object_category/">object_category</a></td>
<td>0..1 <br/> <a href="../OntologyClass/">OntologyClass</a></td>
<td>Used to hold the biolink class/category of an association</td>
<td><a href="../Association/">Association</a></td>
</tr>
<tr>
<td><a href="../subject_closure/">subject_closure</a></td>
<td>0..* <br/> <a href="../String/">String</a></td>
<td>Used to hold the subject closure of an association</td>
<td><a href="../Association/">Association</a></td>
</tr>
<tr>
<td><a href="../object_closure/">object_closure</a></td>
<td>0..* <br/> <a href="../String/">String</a></td>
<td>Used to hold the object closure of an association</td>
<td><a href="../Association/">Association</a></td>
</tr>
<tr>
<td><a href="../subject_category_closure/">subject_category_closure</a></td>
<td>0..* <br/> <a href="../OntologyClass/">OntologyClass</a></td>
<td>Used to hold the subject category closure of an association</td>
<td><a href="../Association/">Association</a></td>
</tr>
<tr>
<td><a href="../object_category_closure/">object_category_closure</a></td>
<td>0..* <br/> <a href="../OntologyClass/">OntologyClass</a></td>
<td>Used to hold the object category closure of an association</td>
<td><a href="../Association/">Association</a></td>
</tr>
<tr>
<td><a href="../subject_namespace/">subject_namespace</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Used to hold the subject namespace of an association</td>
<td><a href="../Association/">Association</a></td>
</tr>
<tr>
<td><a href="../object_namespace/">object_namespace</a></td>
<td>0..1 <br/> <a href="../String/">String</a></td>
<td>Used to hold the object namespace of an association</td>
<td><a href="../Association/">Association</a></td>
</tr>
<tr>
<td><a href="../subject_label_closure/">subject_label_closure</a></td>
<td>0..* <br/> <a href="../String/">String</a></td>
<td>Used to hold the subject label closure of an association</td>
<td><a href="../Association/">Association</a></td>
</tr>
<tr>
<td><a href="../object_label_closure/">object_label_closure</a></td>
<td>0..* <br/> <a href="../String/">String</a></td>
<td>Used to hold the object label closure of an association</td>
<td><a href="../Association/">Association</a></td>
</tr>
<tr>
<td><a href="../retrieval_source_ids/">retrieval_source_ids</a></td>
<td>0..* <br/> <a href="../RetrievalSource/">RetrievalSource</a></td>
<td>A list of retrieval sources that served as a source of knowledge expressed in...</td>
<td><a href="../Association/">Association</a></td>
</tr>
<tr>
<td><a href="../id/">id</a></td>
<td>1..1 <br/> <a href="../String/">String</a></td>
<td>A unique identifier for an entity</td>
<td><a href="../Entity/">Entity</a></td>
</tr>
<tr>
<td><a href="../iri/">iri</a></td>
<td>0..1 <br/> <a href="../IriType/">IriType</a></td>
<td>An IRI for an entity</td>
<td><a href="../Entity/">Entity</a></td>
</tr>
<tr>
<td><a href="../category/">category</a></td>
<td>0..* <br/> <a href="../CategoryType/">CategoryType</a></td>
<td>Name of the high level ontology class in which this entity is categorized</td>
<td><a href="../Entity/">Entity</a></td>
</tr>
<tr>
<td><a href="../type/">type</a></td>
<td>0..* <br/> <a href="../String/">String</a></td>
<td>rdf:type of biolink:Association should be fixed at rdf:Statement</td>
<td><a href="../Entity/">Entity</a></td>
</tr>
<tr>
<td><a href="../name/">name</a></td>
<td>0..1 <br/> <a href="../LabelType/">LabelType</a></td>
<td>A human-readable name for an attribute or entity</td>
<td><a href="../Entity/">Entity</a></td>
</tr>
<tr>
<td><a href="../description/">description</a></td>
<td>0..1 <br/> <a href="../NarrativeText/">NarrativeText</a></td>
<td>a human-readable description of an entity</td>
<td><a href="../Entity/">Entity</a></td>
</tr>
<tr>
<td><a href="../has_attribute/">has_attribute</a></td>
<td>0..* <br/> <a href="../Attribute/">Attribute</a></td>
<td>connects any entity to an attribute</td>
<td><a href="../Entity/">Entity</a></td>
</tr>
<tr>
<td><a href="../has_count/">has_count</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>number of things with a particular property</td>
<td><a href="../FrequencyQuantifier/">FrequencyQuantifier</a></td>
</tr>
<tr>
<td><a href="../has_total/">has_total</a></td>
<td>0..1 <br/> <a href="../Integer/">Integer</a></td>
<td>total number of things in a particular reference set</td>
<td><a href="../FrequencyQuantifier/">FrequencyQuantifier</a></td>
</tr>
<tr>
<td><a href="../has_quotient/">has_quotient</a></td>
<td>0..1 <br/> <a href="../Double/">Double</a></td>
<td></td>
<td><a href="../FrequencyQuantifier/">FrequencyQuantifier</a></td>
</tr>
<tr>
<td><a href="../has_percentage/">has_percentage</a></td>
<td>0..1 <br/> <a href="../Double/">Double</a></td>
<td>equivalent to has quotient multiplied by 100</td>
<td><a href="../FrequencyQuantifier/">FrequencyQuantifier</a></td>
</tr>
<tr>
<td><a href="../severity_qualifier/">severity_qualifier</a></td>
<td>0..1 <br/> <a href="../SeverityValue/">SeverityValue</a></td>
<td>a qualifier used in a phenotypic association to state how severe the phenotyp...</td>
<td><a href="../EntityToFeatureOrDiseaseQualifiersMixin/">EntityToFeatureOrDiseaseQualifiersMixin</a></td>
</tr>
<tr>
<td><a href="../onset_qualifier/">onset_qualifier</a></td>
<td>0..1 <br/> <a href="../Onset/">Onset</a></td>
<td>a qualifier used in a phenotypic association to state when the phenotype appe...</td>
<td><a href="../EntityToFeatureOrDiseaseQualifiersMixin/">EntityToFeatureOrDiseaseQualifiersMixin</a></td>
</tr>
<tr>
<td><a href="../frequency_qualifier/">frequency_qualifier</a></td>
<td>0..1 <br/> <a href="../FrequencyValue/">FrequencyValue</a></td>
<td>a qualifier used in a phenotypic association to state how frequent the phenot...</td>
<td><a href="../FrequencyQualifierMixin/">FrequencyQualifierMixin</a></td>
</tr>
</tbody>
</table>
<h2 id="identifier-and-mapping-information">Identifier and Mapping Information</h2>
<h3 id="schema-source">Schema Source</h3>
<ul>
<li>from schema: https://w3id.org/biolink/biolink-model</li>
</ul>
<h2 id="mappings">Mappings</h2>
<table>
<thead>
<tr>
<th>Mapping Type</th>
<th>Mapped Value</th>
</tr>
</thead>
<tbody>
<tr>
<td>self</td>
<td>biolink:GeneAsAModelOfDiseaseAssociation</td>
</tr>
<tr>
<td>native</td>
<td>biolink:GeneAsAModelOfDiseaseAssociation</td>
</tr>
</tbody>
</table>
<h2 id="linkml-source">LinkML Source</h2>
<!-- TODO: investigate https://stackoverflow.com/questions/37606292/how-to-create-tabbed-code-blocks-in-mkdocs-or-sphinx -->
<h3 id="direct">Direct</h3>
<details>
<pre><code class="language-yaml">name: gene as a model of disease association
from_schema: https://w3id.org/biolink/biolink-model
is_a: gene to disease association
mixins:
- model to disease association mixin
- entity to disease association mixin
slot_usage:
  subject:
    name: subject
    description: A gene that has a role in modeling the disease. This may be a model
      organism ortholog of a known disease gene, or it may be a gene whose mutants
      recapitulate core features of the disease.
    domain_of:
    - association
    range: gene or gene product
defining_slots:
- subject
- predicate
- object

</code></pre>
</details>
<h3 id="induced">Induced</h3>
<details>
<pre><code class="language-yaml">name: gene as a model of disease association
from_schema: https://w3id.org/biolink/biolink-model
is_a: gene to disease association
mixins:
- model to disease association mixin
- entity to disease association mixin
slot_usage:
  subject:
    name: subject
    description: A gene that has a role in modeling the disease. This may be a model
      organism ortholog of a known disease gene, or it may be a gene whose mutants
      recapitulate core features of the disease.
    domain_of:
    - association
    range: gene or gene product
attributes:
  subject aspect qualifier:
    name: subject aspect qualifier
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: aspect qualifier
    domain: association
    alias: subject_aspect_qualifier
    owner: gene as a model of disease association
    domain_of:
    - predicate mapping
    - chemical affects gene association
    - gene affects chemical association
    - gene to disease or phenotypic feature association
    range: GeneOrGeneProductOrChemicalEntityAspectEnum
  object direction qualifier:
    name: object direction qualifier
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: direction qualifier
    domain: association
    alias: object_direction_qualifier
    owner: gene as a model of disease association
    domain_of:
    - predicate mapping
    - gene to disease or phenotypic feature association
    - chemical entity or gene or gene product regulates gene association
    range: DirectionQualifierEnum
  sex qualifier:
    name: sex qualifier
    description: a qualifier used in a phenotypic association to state whether the
      association is specific to a particular sex.
    in_subset:
    - translator_minimal
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: qualifier
    domain: association
    alias: sex_qualifier
    owner: gene as a model of disease association
    domain_of:
    - entity to phenotypic feature association mixin
    range: biological sex
  subject:
    name: subject
    description: A gene that has a role in modeling the disease. This may be a model
      organism ortholog of a known disease gene, or it may be a gene whose mutants
      recapitulate core features of the disease.
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: association slot
    domain: association
    slot_uri: rdf:subject
    alias: subject
    owner: gene as a model of disease association
    domain_of:
    - association
    range: gene or gene product
    required: true
  predicate:
    name: predicate
    description: The relationship to the disease
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: association slot
    domain: association
    slot_uri: rdf:predicate
    alias: predicate
    owner: gene as a model of disease association
    domain_of:
    - predicate mapping
    - association
    subproperty_of: model of
    range: predicate type
    required: true
  object:
    name: object
    description: disease
    examples:
    - value: MONDO:0020066
      description: Ehlers-Danlos syndrome
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: association slot
    domain: association
    slot_uri: rdf:object
    alias: object
    owner: gene as a model of disease association
    domain_of:
    - association
    range: disease
    required: true
  negated:
    name: negated
    description: if set to true, then the association is negated i.e. is not true
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: association slot
    domain: association
    alias: negated
    owner: gene as a model of disease association
    domain_of:
    - association
    range: boolean
  qualifiers:
    name: qualifiers
    local_names:
      ga4gh:
        local_name_source: ga4gh
        local_name_value: annotation qualifier
    description: connects an association to qualifiers that modify or qualify the
      meaning of that association
    deprecated: 'True'
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: association slot
    domain: association
    multivalued: true
    alias: qualifiers
    owner: gene as a model of disease association
    domain_of:
    - association
    range: ontology class
  publications:
    name: publications
    description: One or more publications that report the statement expressed in an  Association,
      or provide information used as evidence supporting this statement.
    comments:
    - 'The notion of a ‘Publication’ is considered broadly to include any  document
      made available for public consumption. It covers journal issues,  individual
      articles, and books - and also things like article pre-prints,  white papers,
      patents, drug labels, web pages, protocol documents, etc. '
    from_schema: https://w3id.org/biolink/biolink-model
    aliases:
    - supporting publications
    - supporting documents
    rank: 1000
    is_a: association slot
    domain: association
    multivalued: true
    alias: publications
    owner: gene as a model of disease association
    domain_of:
    - association
    range: publication
  has evidence:
    name: has evidence
    description: connects an association to an instance of supporting evidence
    from_schema: https://w3id.org/biolink/biolink-model
    exact_mappings:
    - RO:0002558
    rank: 1000
    is_a: association slot
    domain: association
    multivalued: true
    alias: has_evidence
    owner: gene as a model of disease association
    domain_of:
    - association
    range: evidence type
  knowledge source:
    name: knowledge source
    description: An Information Resource from which the knowledge expressed in an
      Association was retrieved, directly or indirectly. This can be any resource
      through which the knowledge passed on its way to its currently serialized form.
      In practice, implementers should use one of the more specific subtypes of this
      generic property.
    from_schema: https://w3id.org/biolink/biolink-model
    close_mappings:
    - pav:providedBy
    rank: 1000
    is_a: association slot
    domain: association
    alias: knowledge_source
    owner: gene as a model of disease association
    domain_of:
    - association
    range: string
  primary knowledge source:
    name: primary knowledge source
    description: The most upstream source of the knowledge expressed in an Association
      that an implementer can identify.  Performing a rigorous analysis of upstream
      data providers is expected; every effort is made to catalog the most upstream
      source of data in this property.  Only one data source should be declared primary
      in any association.  "aggregator knowledge source" can be used to capture non-primary
      sources.
    notes:
    - 'For example: a single ChemicalToGene Edge originally curated by ClinicalTrials.org,
      is aggregated by ChEMBL, then  incorporated into the MolePro KP, then sent via
      TRAPI message to the ARAGORN ARA, and finally sent to  the NCATS ARS. The retrieval
      path for this Edge is as follows:  ARS--retrieved_from--&gt;  ARAGORN  --retrieved_from--&gt;   MolePro  --retrieved_from--&gt;
      ChEMBL --retrieved_from--&gt;  ClinicalTrials.gov The "primary knowledge source"
      for this edge is "infores:clinical-trials-gov".  "infores:chembl" and "infores:molecular_data_provider"
      are listed in the "aggregator knowledge source" property.'
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: knowledge source
    domain: association
    multivalued: false
    alias: primary_knowledge_source
    owner: gene as a model of disease association
    domain_of:
    - association
    range: string
  aggregator knowledge source:
    name: aggregator knowledge source
    description: An intermediate aggregator resource from which knowledge expressed
      in an Association was retrieved downstream of the original source, on its path
      to its current serialized form.
    notes:
    - 'For example, in this Feature Variable Association Edge generated by the Exposure
      Agent’s ICEES KP,  through statistical analysis of clinical and environmental
      data supplied by the UNC Clinical Data Warehouse,  the Edge is passed to the
      Ranking Agent’s ARAGORN ARA,  and then on to the ARS. The retrieval path for
      this Edge is as follows: ARS--retrieved_from--&gt;  ARAGORN  --retrieved_from--&gt;   ICEES
      --supporting_data_from--&gt;  UNC Data Warehouse This example illustrates how to
      represent the source provenance of KP-generated knowledge, including the source
      of  data from which the knowledge was derived.    The "primary knowledge source"
      for this edge is "infores:icees-asthma". A "supporting data source" for this
      KP- generated knowledge is "infores:unc-cdw-health."  The "aggregator knowledge
      source" for this data is "infores:aragorn-ara"'
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: knowledge source
    domain: association
    multivalued: true
    alias: aggregator_knowledge_source
    owner: gene as a model of disease association
    domain_of:
    - association
    range: string
  timepoint:
    name: timepoint
    description: a point in time
    from_schema: https://w3id.org/biolink/biolink-model
    aliases:
    - duration
    rank: 1000
    alias: timepoint
    owner: gene as a model of disease association
    domain_of:
    - geographic location at time
    - exposure event
    - association
    range: time type
  original subject:
    name: original subject
    description: used to hold the original subject of a relation (or predicate) that
      an external knowledge source uses before transformation to match the biolink-model
      specification.
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: association slot
    domain: association
    alias: original_subject
    owner: gene as a model of disease association
    domain_of:
    - association
    range: string
  original predicate:
    name: original predicate
    id_prefixes:
    - RO
    - BSPO
    - SIO
    description: used to hold the original relation/predicate that an external knowledge
      source uses before transformation to match the biolink-model specification.
    from_schema: https://w3id.org/biolink/biolink-model
    aliases:
    - original relation
    - relation
    rank: 1000
    is_a: association slot
    domain: association
    alias: original_predicate
    owner: gene as a model of disease association
    domain_of:
    - association
    range: uriorcurie
  original object:
    name: original object
    description: used to hold the original object of a relation (or predicate) that
      an external knowledge source uses before transformation to match the biolink-model
      specification.
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: association slot
    domain: association
    alias: original_object
    owner: gene as a model of disease association
    domain_of:
    - association
    range: string
  subject category:
    name: subject category
    annotations:
      denormalized:
        tag: denormalized
        value: true
    description: Used to hold the biolink class/category of an association. This is
      a denormalized  field used primarily in the SQL serialization of a knowledge
      graph via KGX.
    examples:
    - value: biolink:Gene
      description: The subject category of the association between the gene 'BRCA1'
        and the disease 'breast cancer' is 'biolink:Gene'.
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: association slot
    domain: association
    multivalued: false
    alias: subject_category
    owner: gene as a model of disease association
    domain_of:
    - association
    range: ontology class
  object category:
    name: object category
    annotations:
      denormalized:
        tag: denormalized
        value: true
    description: Used to hold the biolink class/category of an association. This is
      a denormalized  field used primarily in the SQL serialization of a knowledge
      graph via KGX.
    examples:
    - value: biolink:Disease
      description: The object category of the association between the gene 'BRCA1'
        and the disease 'breast cancer' is 'biolink:Disease'.
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: association slot
    domain: association
    multivalued: false
    alias: object_category
    owner: gene as a model of disease association
    domain_of:
    - association
    range: ontology class
  subject closure:
    name: subject closure
    annotations:
      denormalized:
        tag: denormalized
        value: true
    description: Used to hold the subject closure of an association. This is a denormalized  field
      used primarily in the SQL serialization of a knowledge graph via KGX.
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: association slot
    domain: association
    multivalued: true
    alias: subject_closure
    owner: gene as a model of disease association
    domain_of:
    - association
    range: string
  object closure:
    name: object closure
    annotations:
      denormalized:
        tag: denormalized
        value: true
    description: Used to hold the object closure of an association. This is a denormalized  field
      used primarily in the SQL serialization of a knowledge graph via KGX.
    examples:
    - value: '[''MONDO:0000167'', ''MONDO:0005395'']'
      description: 'The object closure of the association between the gene ''BRCA1''
        and the disease ''breast cancer'' is the set of all diseases that are ancestors
        of ''breast cancer'' in the MONDO ontology.  Note: typically the "subclass
        of" and "part of"  relations are used to construct the closure, but other
        relations may be used as well.'
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: association slot
    domain: association
    multivalued: true
    alias: object_closure
    owner: gene as a model of disease association
    domain_of:
    - association
    range: string
  subject category closure:
    name: subject category closure
    annotations:
      denormalized:
        tag: denormalized
        value: true
    description: Used to hold the subject category closure of an association. This
      is a denormalized  field used primarily in the SQL serialization of a knowledge
      graph via KGX.
    examples:
    - value: '[''biolink:Gene", "biolink:NamedThing'']'
      description: 'The subject category closure of the association between the gene
        ''BRCA1'' and the disease ''breast cancer'' is the set of all biolink classes
        that are ancestors of ''biolink:Gene'' in the biolink model.  Note: typically
        the "subclass of" and "part of"  relations are used to construct the closure,
        but other relations may be used as well.'
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: association slot
    domain: association
    multivalued: true
    alias: subject_category_closure
    owner: gene as a model of disease association
    domain_of:
    - association
    range: ontology class
  object category closure:
    name: object category closure
    annotations:
      denormalized:
        tag: denormalized
        value: true
    description: Used to hold the object category closure of an association. This
      is a denormalized  field used primarily in the SQL serialization of a knowledge
      graph via KGX.
    examples:
    - value: '[''biolink:Disease", "biolink:NamedThing'']'
      description: 'The object category closure of the association between the gene
        ''BRCA1'' and the disease ''breast cancer'' is the set of all biolink classes
        that are ancestors of ''biolink:Disease'' in the biolink model.  Note: typically
        the "subclass of" and "part of"  relations are used to construct the closure,
        but other relations may be used as well.'
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: association slot
    domain: association
    multivalued: true
    alias: object_category_closure
    owner: gene as a model of disease association
    domain_of:
    - association
    range: ontology class
  subject namespace:
    name: subject namespace
    annotations:
      denormalized:
        tag: denormalized
        value: true
    description: Used to hold the subject namespace of an association. This is a denormalized  field
      used primarily in the SQL serialization of a knowledge graph via KGX.
    examples:
    - value: NCBIGene
      description: The subject namespace of the association between the gene 'BRCA1'
        and the disease 'breast cancer' is 'NCBIGene'.
    from_schema: https://w3id.org/biolink/biolink-model
    aliases:
    - subject prefix
    rank: 1000
    is_a: association slot
    domain: association
    multivalued: false
    alias: subject_namespace
    owner: gene as a model of disease association
    domain_of:
    - association
    range: string
  object namespace:
    name: object namespace
    annotations:
      denormalized:
        tag: denormalized
        value: true
    description: Used to hold the object namespace of an association. This is a denormalized  field
      used primarily in the SQL serialization of a knowledge graph via KGX.
    examples:
    - value: MONDO
      description: The object namespace of the association between the gene 'BRCA1'
        and the disease 'breast cancer' is 'MONDO'.
    from_schema: https://w3id.org/biolink/biolink-model
    aliases:
    - object prefix
    rank: 1000
    is_a: association slot
    domain: association
    multivalued: false
    alias: object_namespace
    owner: gene as a model of disease association
    domain_of:
    - association
    range: string
  subject label closure:
    name: subject label closure
    annotations:
      denormalized:
        tag: denormalized
        value: true
    description: Used to hold the subject label closure of an association. This is
      a denormalized  field used primarily in the SQL serialization of a knowledge
      graph via KGX.
    examples:
    - value: '[''BRACA1'']'
      description: The subject label closure of the association between the gene 'BRCA1'
        and the disease 'breast cancer' is the set of all labels that are ancestors
        of 'BRCA1' in the biolink model.
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: association slot
    domain: association
    multivalued: true
    alias: subject_label_closure
    owner: gene as a model of disease association
    domain_of:
    - association
    range: string
  object label closure:
    name: object label closure
    annotations:
      denormalized:
        tag: denormalized
        value: true
    description: Used to hold the object label closure of an association. This is
      a denormalized  field used primarily in the SQL serialization of a knowledge
      graph via KGX.
    examples:
    - value: '[''breast cancer'', ''cancer'']'
      description: The object label closure of the association between the gene 'BRCA1'
        and the disease 'breast cancer' is the set of all labels that are ancestors
        of 'breast cancer' in the biolink model.
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: association slot
    domain: association
    multivalued: true
    alias: object_label_closure
    owner: gene as a model of disease association
    domain_of:
    - association
    range: string
  retrieval source ids:
    name: retrieval source ids
    description: A list of retrieval sources that served as a source of knowledge
      expressed in an Edge, or a source of data used to generate this knowledge.
    in_subset:
    - translator_minimal
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    multivalued: true
    alias: retrieval_source_ids
    owner: gene as a model of disease association
    domain_of:
    - association
    range: retrieval source
  id:
    name: id
    description: A unique identifier for an entity. Must be either a CURIE shorthand
      for a URI or a complete URI
    in_subset:
    - translator_minimal
    from_schema: https://w3id.org/biolink/biolink-model
    exact_mappings:
    - AGRKB:primaryId
    - gff3:ID
    - gpi:DB_Object_ID
    rank: 1000
    domain: entity
    identifier: true
    alias: id
    owner: gene as a model of disease association
    domain_of:
    - ontology class
    - entity
    range: string
    required: true
  iri:
    name: iri
    description: An IRI for an entity. This is determined by the id using expansion
      rules.
    in_subset:
    - translator_minimal
    - samples
    from_schema: https://w3id.org/biolink/biolink-model
    exact_mappings:
    - WIKIDATA_PROPERTY:P854
    rank: 1000
    alias: iri
    owner: gene as a model of disease association
    domain_of:
    - attribute
    - entity
    range: iri type
  category:
    name: category
    description: "Name of the high level ontology class in which this entity is categorized.\
      \ Corresponds to the label for the biolink entity type class.\n * In a neo4j\
      \ database this MAY correspond to the neo4j label tag.\n * In an RDF database\
      \ it should be a biolink model class URI.\nThis field is multi-valued. It should\
      \ include values for ancestors of the biolink class; for example, a protein\
      \ such as Shh would have category values `biolink:Protein`, `biolink:GeneProduct`,\
      \ `biolink:MolecularEntity`, ...\nIn an RDF database, nodes will typically have\
      \ an rdf:type triples. This can be to the most specific biolink class, or potentially\
      \ to a class more specific than something in biolink. For example, a sequence\
      \ feature `f` may have a rdf:type assertion to a SO class such as TF_binding_site,\
      \ which is more specific than anything in biolink. Here we would have categories\
      \ {biolink:GenomicEntity, biolink:MolecularEntity, biolink:NamedThing}"
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: type
    domain: entity
    multivalued: true
    designates_type: true
    alias: category
    owner: gene as a model of disease association
    domain_of:
    - entity
    is_class_field: true
    range: category type
    required: false
  type:
    name: type
    description: rdf:type of biolink:Association should be fixed at rdf:Statement
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    domain: entity
    slot_uri: rdf:type
    multivalued: true
    alias: type
    owner: gene as a model of disease association
    domain_of:
    - entity
    range: string
  name:
    name: name
    description: A human-readable name for an attribute or entity.
    in_subset:
    - translator_minimal
    - samples
    from_schema: https://w3id.org/biolink/biolink-model
    aliases:
    - label
    - display name
    - title
    exact_mappings:
    - gff3:Name
    - gpi:DB_Object_Name
    narrow_mappings:
    - dct:title
    - WIKIDATA_PROPERTY:P1476
    rank: 1000
    domain: entity
    slot_uri: rdfs:label
    alias: name
    owner: gene as a model of disease association
    domain_of:
    - attribute
    - entity
    - macromolecular machine mixin
    range: label type
  description:
    name: description
    description: a human-readable description of an entity
    in_subset:
    - translator_minimal
    from_schema: https://w3id.org/biolink/biolink-model
    aliases:
    - definition
    exact_mappings:
    - IAO:0000115
    - skos:definitions
    narrow_mappings:
    - gff3:Description
    rank: 1000
    slot_uri: dct:description
    alias: description
    owner: gene as a model of disease association
    domain_of:
    - entity
    range: narrative text
  has attribute:
    name: has attribute
    description: connects any entity to an attribute
    in_subset:
    - samples
    from_schema: https://w3id.org/biolink/biolink-model
    exact_mappings:
    - SIO:000008
    close_mappings:
    - OBI:0001927
    narrow_mappings:
    - OBAN:association_has_subject_property
    - OBAN:association_has_object_property
    - CPT:has_possibly_included_panel_element
    - DRUGBANK:category
    - EFO:is_executed_in
    - HANCESTRO:0301
    - LOINC:has_action_guidance
    - LOINC:has_adjustment
    - LOINC:has_aggregation_view
    - LOINC:has_approach_guidance
    - LOINC:has_divisor
    - LOINC:has_exam
    - LOINC:has_method
    - LOINC:has_modality_subtype
    - LOINC:has_object_guidance
    - LOINC:has_scale
    - LOINC:has_suffix
    - LOINC:has_time_aspect
    - LOINC:has_time_modifier
    - LOINC:has_timing_of
    - NCIT:R88
    - NCIT:eo_disease_has_property_or_attribute
    - NCIT:has_data_element
    - NCIT:has_pharmaceutical_administration_method
    - NCIT:has_pharmaceutical_basic_dose_form
    - NCIT:has_pharmaceutical_intended_site
    - NCIT:has_pharmaceutical_release_characteristics
    - NCIT:has_pharmaceutical_state_of_matter
    - NCIT:has_pharmaceutical_transformation
    - NCIT:is_qualified_by
    - NCIT:qualifier_applies_to
    - NCIT:role_has_domain
    - NCIT:role_has_range
    - INO:0000154
    - HANCESTRO:0308
    - OMIM:has_inheritance_type
    - orphanet:C016
    - orphanet:C017
    - RO:0000053
    - RO:0000086
    - RO:0000087
    - SNOMED:has_access
    - SNOMED:has_clinical_course
    - SNOMED:has_count_of_base_of_active_ingredient
    - SNOMED:has_dose_form_administration_method
    - SNOMED:has_dose_form_release_characteristic
    - SNOMED:has_dose_form_transformation
    - SNOMED:has_finding_context
    - SNOMED:has_finding_informer
    - SNOMED:has_inherent_attribute
    - SNOMED:has_intent
    - SNOMED:has_interpretation
    - SNOMED:has_laterality
    - SNOMED:has_measurement_method
    - SNOMED:has_method
    - SNOMED:has_priority
    - SNOMED:has_procedure_context
    - SNOMED:has_process_duration
    - SNOMED:has_property
    - SNOMED:has_revision_status
    - SNOMED:has_scale_type
    - SNOMED:has_severity
    - SNOMED:has_specimen
    - SNOMED:has_state_of_matter
    - SNOMED:has_subject_relationship_context
    - SNOMED:has_surgical_approach
    - SNOMED:has_technique
    - SNOMED:has_temporal_context
    - SNOMED:has_time_aspect
    - SNOMED:has_units
    - UMLS:has_structural_class
    - UMLS:has_supported_concept_property
    - UMLS:has_supported_concept_relationship
    - UMLS:may_be_qualified_by
    rank: 1000
    domain: entity
    multivalued: true
    alias: has_attribute
    owner: gene as a model of disease association
    domain_of:
    - entity
    range: attribute
  has count:
    name: has count
    description: number of things with a particular property
    from_schema: https://w3id.org/biolink/biolink-model
    exact_mappings:
    - LOINC:has_count
    rank: 1000
    is_a: aggregate statistic
    domain: named thing
    alias: has_count
    owner: gene as a model of disease association
    domain_of:
    - frequency quantifier
    range: integer
  has total:
    name: has total
    description: total number of things in a particular reference set
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: aggregate statistic
    domain: named thing
    alias: has_total
    owner: gene as a model of disease association
    domain_of:
    - frequency quantifier
    range: integer
  has quotient:
    name: has quotient
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: aggregate statistic
    domain: named thing
    alias: has_quotient
    owner: gene as a model of disease association
    domain_of:
    - frequency quantifier
    range: double
  has percentage:
    name: has percentage
    description: equivalent to has quotient multiplied by 100
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: aggregate statistic
    domain: named thing
    alias: has_percentage
    owner: gene as a model of disease association
    domain_of:
    - frequency quantifier
    range: double
  severity qualifier:
    name: severity qualifier
    description: a qualifier used in a phenotypic association to state how severe
      the phenotype is in the subject
    in_subset:
    - translator_minimal
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: qualifier
    domain: association
    alias: severity_qualifier
    owner: gene as a model of disease association
    domain_of:
    - entity to feature or disease qualifiers mixin
    range: severity value
  onset qualifier:
    name: onset qualifier
    description: a qualifier used in a phenotypic association to state when the phenotype
      appears is in the subject
    in_subset:
    - translator_minimal
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: qualifier
    domain: association
    alias: onset_qualifier
    owner: gene as a model of disease association
    domain_of:
    - entity to feature or disease qualifiers mixin
    range: onset
  frequency qualifier:
    name: frequency qualifier
    description: a qualifier used in a phenotypic association to state how frequent
      the phenotype is observed in the subject
    in_subset:
    - translator_minimal
    from_schema: https://w3id.org/biolink/biolink-model
    rank: 1000
    is_a: qualifier
    domain: association
    alias: frequency_qualifier
    owner: gene as a model of disease association
    domain_of:
    - frequency qualifier mixin
    range: frequency value
defining_slots:
- subject
- predicate
- object

</code></pre>
</details>
</article>
</div>
<script>var tabs=__md_get("__tabs");if(Array.isArray(tabs))e:for(var set of document.querySelectorAll(".tabbed-set")){var tab,labels=set.querySelector(".tabbed-labels");for(tab of tabs)for(var label of labels.getElementsByTagName("label"))if(label.innerText.trim()===tab){var input=document.getElementById(label.htmlFor);input.checked=!0;continue e}}</script>
</div>
</main>
<footer class="md-footer">
<div class="md-footer-meta md-typeset">
<div class="md-footer-meta__inner md-grid">
<div class="md-copyright">
  
  
    Made with
    <a href="https://squidfunk.github.io/mkdocs-material/" rel="noopener" target="_blank">
      Material for MkDocs
    </a>
</div>
</div>
</div>
</footer>
</div>
<div class="md-dialog" data-md-component="dialog">
<div class="md-dialog__inner md-typeset"></div>
</div>
<script id="__config" type="application/json">{"base": "..", "features": ["content.tabs.link"], "search": "../assets/javascripts/workers/search.16e2a7d4.min.js", "translations": {"clipboard.copied": "Copied to clipboard", "clipboard.copy": "Copy to clipboard", "search.config.lang": "en", "search.config.pipeline": "trimmer, stopWordFilter", "search.config.separator": "[\\s\\-]+", "search.placeholder": "Search", "search.result.more.one": "1 more on this page", "search.result.more.other": "# more on this page", "search.result.none": "No matching documents", "search.result.one": "1 matching document", "search.result.other": "# matching documents", "search.result.placeholder": "Type to start searching", "search.result.term.missing": "Missing", "select.version.title": "Select version"}}</script>
<script src="../assets/javascripts/bundle.5a2dcb6a.min.js"></script>
<script src="https://unpkg.com/mermaid@8.8.0/dist/mermaid.min.js"></script><script>mermaid.initialize({});</script></body>
</html>